client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -90,6 +90,9 @@
     @Option(names = "--truststore-type", paramLabel = "<type>", description = "Truststore type")
     public Optional<String> truststoreType;
 
+    @Option(names = "--truststore-use-system-store", description = "Use system truststore")
```

Comment:
system as in JVM .. or as in OS .. or both?

Reply:
also .. needs to be added to the docs

Reply:
I updated the documentation, don't know if I need to further expand though